frontend/assets/javascripts/src/modules/form/validation/listeners.js

```diff
@@ -36,12 +35,8 @@ define([
             });
 
             if (!form.errs.length) {
-                if (form.hasPayment) {
-                    processingMessage.replace('...', ' payment...');
```

Comment:
This is bit janky so have removed, doesn't need to replace anything just "Processing…" is fine at this stage.